packages/fetch/test/fetchFromRegistry.ts

```diff
@@ -1,76 +1,72 @@
 /// <reference path="../../../typings/index.d.ts"/>
 import { createFetchFromRegistry } from '@pnpm/fetch'
 import nock = require('nock')
-import test = require('tape')
 
-test('fetchFromRegistry', async t => {
-  const fetchFromRegistry = createFetchFromRegistry({})
-  const res = await fetchFromRegistry('https://registry.npmjs.org/is-positive')
-  const metadata = await res.json()
-  t.equal(metadata.name, 'is-positive')
-  t.notOk(metadata.versions['1.0.0'].scripts)
-  t.end()
-})
-
-test('fetchFromRegistry fullMetadata', async t => {
-  const fetchFromRegistry = createFetchFromRegistry({ fullMetadata: true })
-  const res = await fetchFromRegistry('https://registry.npmjs.org/is-positive')
-  const metadata = await res.json()
-  t.equal(metadata.name, 'is-positive')
-  t.ok(metadata.versions['1.0.0'].scripts)
-  t.end()
-})
+describe('fetch', () => {
```

Comment:
I think to reduce the number of changes, might be good to omit the describe() for now. Just using test() functions work with Jest as well

Reply:
I thought describe was mandatory. Or it isn't?

Reply:
An, it isn't 👍 ..  I didn't know. I'll remove them.